### PR TITLE
Extract inline styles from contentBlock

### PR DESCRIPTION
**Pull request by pdillon**
Addresses #153 by extracting the style key strings from the content block.

### CODE REVIEW

File: src/model/encoding/encodeInlineStyleRanges.js [hunk 1]
```diff
@@ -68,7 +68,10 @@ function encodeInlineStyleRanges(
   block: ContentBlock
 ): Array<InlineStyleRange> {
   var styleList = block.getCharacterList().map(c => c.getStyle()).toList();
-  var styles = Object.keys(DefaultDraftInlineStyle);
+  var styles = styleList
+    .flatten()
```
Comment (hellendag) [inline src/model/encoding/encodeInlineStyleRanges.js]: Did you find `flatten` to be necessary? The List should already be flat.

File: src/model/encoding/encodeInlineStyleRanges.js [hunk 2]
```diff
@@ -68,7 +68,10 @@ function encodeInlineStyleRanges(
   block: ContentBlock
 ): Array<InlineStyleRange> {
   var styleList = block.getCharacterList().map(c => c.getStyle()).toList();
```
Comment (hellendag) [inline src/model/encoding/encodeInlineStyleRanges.js]: Can the `toList` step be skipped, since it's going to become a `Set`?

**Comment by hellendag** [comment]
Nice!

File: src/model/encoding/encodeInlineStyleRanges.js (hunk 1 above)
Comment (pdillon) [inline src/model/encoding/encodeInlineStyleRanges.js]: Yeah, styleList is a List of OrderedSet.

Comment (hellendag) [inline src/model/encoding/encodeInlineStyleRanges.js]: Oh of course!

**Comment by hellendag** [comment]
@facebook-github-bot import

**Comment by hellendag** [comment]
About to sync this back to merge it. I made a couple of minor changes and added a test spec. :)

Thanks for the PR!

**Comment by pdillon** [comment]
Nice! Thanks and thanks again for draft!

**Pull request closed without merging**